index.md

```diff
@@ -17,8 +17,8 @@ large number of consumers. Pulp can locally mirror all or part of a repository,
 software packages in repositories, and manage many types of content from multiple sources in one
 place.
 
-Pulp has a [REST API](https://docs.pulpproject.org/dev-guide/integration/rest-api/index.html) and
-[command line interface](https://docs.pulpproject.org/user-guide/admin-client/index.html) for
+Pulp has a [REST API](https://docs.pulpproject.org/en/2.21/dev-guide/integration/rest-api/index.html) and
```

Comment:
If we want Pulp 3 api here, do we want:
a) https://docs.pulpproject.org/rest_api.html
or
b) https://docs.pulpproject.org/restapi.html

Reply:
I think the former since there might be more REST API related information that we might add later.